Index: 第八章 Python文件操作/01_文件编码概念.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/第八章 Python文件操作/01_文件编码概念.py b/第八章 Python文件操作/01_文件编码概念.py
new file mode 100644
--- /dev/null	(date 1680057798627)
+++ b/第八章 Python文件操作/01_文件编码概念.py	(date 1680057798627)
@@ -0,0 +1,9 @@
+"""
+编码
+1. 什么是编码 ?
+编码是一种规则集合记录了内容和二进制间进行相互转换的逻辑.
+编码中有许多种,我们最常用的是UTF-8编码
+2. 为什么需要使用编码 ?
+计算机只认识0和1, 所以需要将内容翻译成0和1才能保存在计算机中.
+同时也需要编码, 将计算机保存的O和1, 反向翻译回可以识别的内容.
+"""
\ No newline at end of file
