Index: 第九章 异常、模块与包/03_异常的传递性.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/第九章 异常、模块与包/03_异常的传递性.py b/第九章 异常、模块与包/03_异常的传递性.py
new file mode 100644
--- /dev/null	(date 1680485002205)
+++ b/第九章 异常、模块与包/03_异常的传递性.py	(date 1680485002205)
@@ -0,0 +1,23 @@
+"""
+演示异常的传递性
+"""
+
+# 定义一个出现异常的方法
+def func1():
+    print("func1 开始执行")
+    num = 1 / 0     #肯定有异常,除以0的异常
+    print("func1 结束执行")
+
+# 定义一个无异常的方法,调用上面的方法
+def func2():
+    print("func2 开始执行")
+    func1()
+    print("func2 结束执行")
+
+# 定义一个方法,调用上面的方法
+def main():
+    try:
+        func2()
+    except Exception as e:
+        print(f"出现异常了, 异常的信息是:{e}")
+main()
